doc/changes/changes48.xml

```diff
@@ -0,0 +1,142 @@
+<Chapter Label="ChangesGAP47toGAP48"> 
+<Heading>Changes between &GAP; 4.7 and &GAP; 4.8</Heading>
+
+This chapter contains an overview of most important changes 
+introduced in &GAP; 4.8.0 release (the beta release of &GAP; 4.8).
+Later it will also contain information about subsequent update 
+releases for &GAP; 4.8.
+
+First of all, &GAP; development repository is now hosted on GitHub at
+<URL>https://github.com/gap-system/gap</URL>, and &GAP; 4.8 is the first 
+major &GAP; release made from this repository. The public issue tracker
+for the core &GAP; system is located at 
+<URL>https://github.com/gap-system/gap/issues</URL>,
+and you may use appropriate milestones from 
+<URL>https://github.com/gap-system/gap/milestones</URL> to see all changes
+that were introduced in corresponding &GAP; releases. An overview of the
+most significant ones is provided below.
+
+<Section Label="gap480"> 
+<Heading>&GAP; 4.8.0 (Autumn 2015)</Heading>
+
+<Subsection Label="Changes in the core GAP48 system"> 
+<Heading>Changes in the core &GAP; system introduced in &GAP; 4.8</Heading>
+
+Improved and extended functionality:
+<List>
+<Item>
+Unnecessary tests for the finiteness of the field were removed from 
+<Ref Func="GModuleByMats" BookName="ref"/> and related functions 
+(those that do not need MeatAxe, but just linear algebra).
+Thus the correponding functions will work in the case of characteristic 0
+or for other infinite fields, too.
+</Item> 
+<!-- https://github.com/gap-system/gap/pull/192 -->
+<Item>Extended support for accessing lists using multiple indices and
+indexing into lists using indices other than positive small integers.
```

Comment:
I meant to say that the ways to access lists have been extended. Will reformulate.